engine/src/main/resources/templates/OCM/clusterUpdateInstantEmailBodyV2.html

```diff
@@ -0,0 +1,46 @@
+{@boolean renderSection1=true}
+{@boolean renderSection2=true}
+{#include Common/insightsEmailBody}
+{#content-title}
+    Cluster Manager - OpenShift
+{/content-title}
+{#content-title-section1}
+    {#if action.events[0].payload.global_vars.upgrade_status == 'scheduled'}Upgrade scheduled{#else}Upgrade ended{/if}
+{/content-title-section1}
+{#content-body-section1}
+{#let global_var=action.events[0].payload.global_vars}
+<p>
+This notification is for your <a href="https://cloud.redhat.com/openshift/details/s/{action.events[0].payload.subscription_id}#overview" target="_blank" title="*|CLUSTER_DISPLAY_NAME|*">{action.events[0].payload.cluster_display_name} cluster</a>.
```

Comment:
Note for later: we'll need a way to make all URLs work with both stage and prod.